Microsoft.Toolkit.Uwp.SampleApp/SamplePages/RangeSelector/RangeSelectorCode.bind

```diff
@@ -24,10 +24,14 @@
                  Foreground="Black"
                  Text="{Binding RangeMin, ElementName=RangeSelectorControl, Converter={StaticResource StringFormatConverter}, ConverterParameter='{}{0:0.##}'}" />
       <controls:RangeSelector x:Name="RangeSelectorControl"
-                              Grid.Column="1"                              
-                              Minimum="@[Minimum:Slider:0:0-100]@"
-                              Maximum="@[Maximum:Slider:100:0-100]@"
-                              StepFrequency="@[StepFrequency:DoubleSlider:1.0:0.25-10.0]@"/>
+                              Grid.Column="1"
+                              Minimum="@[Minimum:Slider:0:0-100]"
+                              Maximum="@[Maximum:Slider:100:0-100]"
+                              StepFrequency="@[StepFrequency:DoubleSlider:1.0:0.25-10.0]"
```

Comment:
Could you change these to regular (integer) sliders? Seems like how these properties would be used more often.

Reply:
I'd be ok if we could even limit it to .5 increments

Reply:
should we go with .5 or integers?

Reply:
Let's go with 0.5